benchmarks/sparse/benchmark_semi_structured_sparsity.py

```diff
@@ -62,7 +62,7 @@ def test_linear(m, k, n, dtype, contiguous, backend):
     dense_output = model(input_tensor)
 
     # sparsify weights
-    model.linear.weight = nn.Parameter(to_sparse_semi_structured(sparse_weight, mask=mask.bool()))
+    model.linear.weight = nn.Parameter(SparseSemiStructuredTensor(sparse_weight,))
```

Comment:
Please don't use the class constructor. The factory function is the interface we want to support. This is similar to the to_sparse_csr etc. conversion methods we currently have.

Reply:
Fixed, it was an oversight (initially changed all `to_sparse_semi_structured()` calls to using constructor, then reverted, apparently forgot to revert this one).